packages/build/tests/plugins/fixtures/git_no_root/netlify.toml

```diff
@@ -1,2 +1,2 @@
 [[plugins]]
-package = "./plugin"
+package = "./plugin.mjs"
```

Comment:
Some of the tests are replicating the case where a site does not have a `package.json`. We do this by copying the test fixture to the global temporary directory.
Since there is no `package.json`, we need to rely on `.mjs` for those tests.